readthedocs/config/config.py

```diff
@@ -514,7 +513,7 @@ def validate_python(self):
             if 'version' in raw_python:
                 with self.catch_validation_error('python.version'):
                     python['version'] = validate_choice(
-                        raw_python['version'],
+                        str(raw_python['version']),
```

Comment:
It feels like we're hiding this away here, we should probably make it more obvious that we're converted to a string by at least making it a variable above the `validate_choice` function. Do we need to handle any exceptions that will happen from calling `str`, or are there not any?

Reply:
> Do we need to handle any exceptions that will happen from calling str, or are there not any?

Don't think so, all objects inherit from object and that one has `__str__`